File: POO/Exo1/Implementation C++/AlimentPrincipal.cpp
#include <iostream>
#include "AlimentPrincipal.hpp"

using namespace std;

AlimentPrincipal::AlimentPrincipal(string NomAliment,string TextureAliment,string CouleurAliment)	{
		mNomAliment=NomAliment;
		mTextureAliment=TextureAliment;
		mCouleurAliment=CouleurAliment;
	}
string AlimentPrincipal::AfficherNomAliment(){
		return mNomAliment;
	}
void AlimentPrincipal::ModifierNomAliment(string NomAliment){
		mNomAliment=NomAliment;
	}
string AlimentPrincipal::AfficherTextureAliment(){
		return mTextureAliment;
	}
string AlimentPrincipal::AfficherCouleurAliment(){
		return mCouleurAliment;
	}

File: POO/Exo1/implementation java/AlimentPrincipal.java
public class AlimentPrincipal{
	private String mNomAliment;
	private String mTextureAliment;
	private String mCouleurAliment;
	public AlimentPrincipal(String NomAliment, String TextureAliment, String CouleurAliment)
	{
		this.mNomAliment=NomAliment;
		this.mTextureAliment=TextureAliment;
		this.mCouleurAliment=CouleurAliment;
	}
	public String AfficherNomAliment()
	{
		return this.mNomAliment;
	}
	public void ModifierNomAliment(String NomAliment)
	{
		this.mNomAliment=NomAliment;
	}
	public String AfficherTextureAliment()
	{
		return this.mTextureAliment;
	}
	public String AfficherCouleurAliment()
	{
		return this.mCouleurAliment;
	}
}
